### ISSUE TITLE
Request to implement new $env meta prefix

### ISSUE THREAD

**Issue by XSven** [preaction/Beam-Wire#82]
I have no idea how to refer to environment variables (`%ENV`) in the `args` of a service configuration. It would help to have a new meta prefix `$env`:
```
args:
  user:
    $env: USER
```
should assign `$ENV{USER}` to the named argument `user`.  I would suggest that `$env` refers to a cached (immutable) copy of `%ENV`. If you know a CPAN module that wraps `%ENV` and can be configured as a service to achieve the same please let me know.

**Comment by preaction** [comment]
Having a `$env` meta would be a good idea, I think. There'd be a lot of use, and requiring the indirection of a service or something would be clumsy.

**Comment by XSven** [comment]
Do you accept a PR for this feature too?

**Comment by preaction** [comment]
Yep, I'd accept a PR for this as well. And I've basically left IRC, so issues/e-mail are the best ways to get ahold of me personally at least.
